src/server/v2.0/handler/quota.go

```diff
@@ -63,6 +63,9 @@ func (qa *quotaAPI) ListQuotas(ctx context.Context, params operation.ListQuotasP
 		return qa.SendError(ctx, err)
 	}
 
+	if params.Sort != nil {
+		query.Sorting = *params.Sort
```

Comment:
Why we need to assign the sort again, the line 61 has already pass the sort when build the query, shouldn't handle this in the `BuildQuery`?